src/update/orderbook_rewards.py

```diff
@@ -0,0 +1,50 @@
+"""Pushes a new dune user generated view per Accounting Period
+with name dune_user_generated.cow_rewards_{{PeriodHash}}
+"""
+from duneapi.api import DuneAPI
+from duneapi.types import DuneQuery, Network
+from duneapi.util import open_query
+from pandas import DataFrame, Series
+
+from src.models import AccountingPeriod
+
+
+def dune_repr(df_row: Series) -> str:
+    """
+    This is the per row format for inserting entries in a table of VALUES
+    Specific to a COW Reward Item
+    """
+    return (
+        f"('{df_row['receiver'].lower()}', {df_row['num_trades']}, {df_row['amount']})"
+    )
+
+
+def rewards_df_to_dune_list(data: DataFrame) -> str:
+    """Joins a list of VALUES into the format for postgres VALUES list"""
+    return ",\n".join([dune_repr(row) for _, row in data.iterrows()])
```

Comment:
```suggestion
    data["receiver"] = data["receiver"].apply(lambda x: x.lower())
    return ",\n".join([dune_repr(row) for _, row in data.iterrows()])
```

What do you think about doing this for the entire column at once instead of per row? 

I've also though about trying to do this in a vectorized manner but it isn't as elegant as your solution I believe. Moreover, for small number of rows I don't think this makes a meaningful difference:

```python
    engine = create_engine('sqlite://', echo=False)
    data["receiver"] = data["receiver"].apply(lambda x: x.lower())
    data[['receiver', 'num_trades', 'amount']].to_sql('solvers', con=engine, index=False)
    rows = engine.execute("SELECT * FROM solvers").fetchall()
    return "".join([f"{row},\n" for row in rows])
```

Reply:
I also like this!

Reply:
> I also like this!

:rocket:

Reply:
Is there any particular reason you converted them to "lower"? This isn't Dune V2...

Reply:
Continued discussion [here](https://github.com/cowprotocol/solver-rewards/pull/106/files#r1001772914).